trace-java-client/README.md

```diff
@@ -0,0 +1 @@
+`trace-java-client` is built based on the repo: https://github.com/Omnition/synthetic-load-generator
```

Comment:
It seems it is only used for load generation, then can we move it into existing load generator folder? Or it is expected to be used by some other places?

Reply:
It is also used by the `jaeger-zipkin-sample-app` for end 2 end testing

Reply:
Can you mention packages that depends on trace-java-client in the README?